data/translations/es/state-data.json

```diff
@@ -542,39 +545,39 @@
     "registration_link_language_selection": "true",
     "more_info_link": "https://www.sos.wa.gov/elections/voters/",
     "more_info_link_language_selection": "true",
-    "ip_deadline": "hasta el mismo día de las elecciones",
-    "online_deadline": "hasta 8 días antes del día de las elecciones",
-    "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo 8 días antes del día de las elecciones",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "lunes, 31 de octubre de 2022",
+    "mail_received_deadline": "a más tardar debe ser recibida lunes, 31 de octubre de 2022",
+    "mail_postmarked_deadline": "",
```

Comment:
Updated from postmarked to received. Please check this. Also remove empty deadline properties.